jdbc/composite-ids/pom.xml

```diff
@@ -0,0 +1,35 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<artifactId>spring-data-jdbc-composite-ids</artifactId>
+
+	<parent>
+		<groupId>org.springframework.data.examples</groupId>
+		<artifactId>spring-data-jdbc-examples</artifactId>
+		<version>2.0.0.BUILD-SNAPSHOT</version>
+		<relativePath>../pom.xml</relativePath>
+	</parent>
+
+	<name>Spring Data JDBC - Examples using composite ids</name>
+	<description>Sample project demonstrating Spring Data JDBCs support for custom ids</description>
+
+	<dependencies>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-web</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.data</groupId>
+			<artifactId>spring-data-jdbc</artifactId>
+			<version>4.0.0-SNAPSHOT</version>
```

Comment:
Nit: Use `2025.1.0-M4` BOM, see https://github.com/spring-projects/spring-data-examples/blob/main/jpa/aot-optimization/pom.xml#L18 (also, feel free to bump the JPA and Mongo AOT examples to M4)